workflows/prognostic_c48_run/runtime/names.py

```diff
@@ -20,3 +23,4 @@
     "dQu": EAST_WIND,
     "dQv": NORTH_WIND,
 }
+NUDGING_TENDENCY_SUFFIX = "tendency_due_to_nudging"
```

Comment:
I understood you did this to disambiguate between state updates and tendencies in `add_tendency`, but his new global value introduces a fair amount of coupling between `loop.py`, `diagnostics`, `nudging`, and `test_nudging` and requires more name/string modifying logic. Can you implement this instead by naming the nudging tendencies `dQu`, `dQv`, dQ1, and dQ2 as the "ML" Stepper does? It seems that the existing convention should suffice.

Reply:
If that doesn't work, we can discuss other ways to remove the coupling.

Reply:
I removed the suffix import, and now the nudging tendencies follow the same naming as the ML tendencies.